### PR TITLE
Handle tapping checkmark button.

### PR DESCRIPTION
**Pull request by hieuvuong2310**

### CODE REVIEW

File: ToDoApp/ToDoApp/ToDoListViewModel.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ final class ToDoListViewModel: ObservableObject {
     }
     @Published private(set) var destination: Destination?
     @Published private(set) var state: State = .idle
-    private let taskService: ToDoService
+    @Published private(set) var taskService: ToDoService
```
Comment (vladyslavsosiuk) [inline ToDoApp/ToDoApp/ToDoListViewModel.swift]: Why do we need task service to be a var, as well why do we need it to be published? View doesn't need to observe changes of reference to the Todo service

File: ToDoApp/ToDoApp/ToDoListViewModel.swift [hunk 2]
```diff
@@ -81,4 +81,11 @@ final class ToDoListViewModel: ObservableObject {
             }
         }
     }
+    // Update the status of the task
+    func updateStatus(id: UUID) {
+        Task {
+            _ = await taskService.updateTaskStatus(id: id)
+//            self.fetchToDoTasks()
```
Comment (vladyslavsosiuk) [inline ToDoApp/ToDoApp/ToDoListViewModel.swift]: It seems that this shouldn't be commented

File: ToDoApp/ToDoApp/ToDoService.swift [hunk 3]
```diff
@@ -59,6 +60,7 @@ final class FeaturesToDo: ToDoService {
     }
     convenience init() {
         self.init(dateChecker: Calendar.current)
+        print(tasks)
```
Comment (vladyslavsosiuk) [inline ToDoApp/ToDoApp/ToDoService.swift]: Is this print really needed?

**Review by vladyslavsosiuk — CHANGES_REQUESTED** [review]

File: ToDoApp/ToDoApp/ToDoListViewModel.swift (hunk 2 above)
Comment (hieuvuong2310) [reply ToDoApp/ToDoApp/ToDoListViewModel.swift]: oh my bad, I commented it to debug the code but forgot to uncomment, and it's fixed now.

File: ToDoApp/ToDoApp/ToDoService.swift (hunk 3 above)
Comment (hieuvuong2310) [reply ToDoApp/ToDoApp/ToDoService.swift]: no it's just when I debug the code, but I forgot to delete it, I've fixed it.

File: ToDoApp/ToDoApp/ToDoListViewModel.swift (hunk 1 above)
Comment (hieuvuong2310) [reply ToDoApp/ToDoApp/ToDoListViewModel.swift]: it's fixed now, I just changed it to see whether it caused the bug I had, but the bug was fixed and not related to this.

**Review by vladyslavsosiuk — APPROVED** [review]

**Pull request merged**